backend/api/models.py

```diff
@@ -0,0 +1,59 @@
+from pydantic import BaseModel
+
+
+class GoogleLoginData(BaseModel):
+    token: str
+
+
+class PostSessionResponse(BaseModel):
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class GetUsersResponse(BaseModel):
+    id: int
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class GetCurrentUserResponse(BaseModel):
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class PostUserRequest(BaseModel):
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class PostUserResponse(BaseModel):
+    id: int
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class PutUserRequest(BaseModel):
+    id: int
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class PutUserResponse(BaseModel):
+    id: int
+    email: str
+    is_manager: bool
+    is_admin: bool
+
+
+class DeletedUserResponse(BaseModel):
+    message: str
+
+
+class DeletedSessionResponse(BaseModel):
+    message: str
```

Comment:
These models don't seem to be used currently. Did you mean to update the handlers to return objects?

Reply:
No, I need to delete those models. Thanks for the catch.